scripts/xtensa-build-all.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-SUPPORTED_PLATFORMS=(byt cht bdw hsw apl cnl sue icl skl kbl)
+SUPPORTED_PLATFORMS=(byt cht bdw hsw apl cnl sue icl skl kbl imx)
 BUILD_ROM=no
```

Comment:
Here the platform should be imx8. We will later have a platform named imx8m.

From DSP point of view the platforms are identifcal only the integration is different (e.g imx8/imx8m have different address space mappings for DSP).

Reply:
@dbaluta 
OK, got it as they will use the same compiler?